src/compas/datastructures/mesh/operations/explode.py

```diff
@@ -0,0 +1,90 @@
+__author__     = ['Robin Oval']
+__copyright__  = 'Copyright 2018, Block Research Group - ETH Zurich'
+__license__    = 'MIT License'
+__email__      = 'oval@arch.ethz.ch'
+
+__all__ = [
+    'mesh_disjointed_parts',
+    'mesh_explode'
+]
+
+def mesh_disjointed_parts(mesh):
+    """Get the disjointed parts in a mesh as lists of faces.
+
```

Comment:
what is the difference compared to `compas.topology.connected_components`?

Reply:
done